src/SoftWire.h

```diff
@@ -7,6 +7,32 @@
 #include <stdint.h>
 #include <AsyncDelay.h>
 
+//
+// 2022-04-13 by Technik Gegg
```

Comment:
This is not necessary - it's just adds noise. git handles contributor and date information, and ensures it is correct.

Reply:
Removed